cmd/nerdctl/volume.go

```diff
@@ -44,6 +44,6 @@ func newVolumeCommand() *cobra.Command {
 
 // getVolumeStore returns a volume store
 // that corresponds to a directory like `/var/lib/nerdctl/1935db59/volumes/default`
-func getVolumeStore(globalOptions *types.GlobalCommandOptions) (volumestore.VolumeStore, error) {
+func getVolumeStore(globalOptions types.GlobalCommandOptions) (volumestore.VolumeStore, error) {
 	return volume.Store(globalOptions.Namespace, globalOptions.DataRoot, globalOptions.Address)
 }
```

Comment:
I suggest we remove this function and use `volume.Store` (the 2nd one below) directly (we can change `volume.Store` to accept a globalOptions) (Don't necessarily be part of this PR though)

This adds one unnecessary wrap on volume create:

https://github.com/containerd/nerdctl/blob/40d341dac245a70a6ee4ac43567f2d2818fe3167/cmd/nerdctl/volume.go#L47-L49

https://github.com/containerd/nerdctl/blob/40d341dac245a70a6ee4ac43567f2d2818fe3167/pkg/cmd/volume/volume.go#L26-L32

https://github.com/containerd/nerdctl/blob/40d341dac245a70a6ee4ac43567f2d2818fe3167/pkg/mountutil/volumestore/volumestore.go#L42

Reply:
This should be another PR